src/components/luminave-server/luminave-server-subscription-animation.js

```diff
@@ -126,11 +126,14 @@ class LuminaveServerSubscriptionAnimation extends ApolloSubscription {
         break
 
       case 'shake':
+        // Loop the same 5 seconds of random values
+        duration = 5000
```

Comment:
The behavior when using the duration from Thorium isn't ideal. For long durations, instead of randomly shaking the lights quickly it increases the duration between light changes. Since Luminave will continue looping the shake animation for as long as the animation is active, and since Thorium is in charge of changing the animation from 'shake' to 'normal mode', having Luminave just loop 5 seconds of random shake values provides a fast shake animation while still honoring the actual duration of the shake set in Thorium.

Does that make sense?